phpBB/phpbb/search/exception/no_action_in_progress_exception.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\search\exception;
+
+class no_action_in_progress_exception extends search_exception
```

Comment:
I actually wonder: do we use these exception types anywhere? Having them does not really hurt but I'm also wondering whether we actually need them. ;)

Reply:
I can just launch a search_exception, but is a way to difference between them when displaying errors for example